test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -436,16 +455,6 @@ ava('.getNextWaypointPositionModel() returns the `StaticPositionModel` for the n
 
 ava.todo('.replaceDepartureProcedure() updates the current runway assignment');
 
-ava('.replaceDepartureProcedure() returns early if the nextRouteString matches the current route', (t) => {
```

Comment:
covered not here, but in freshly added test for early return from `.setArrivalRunway()` and `.setDepartureRunway()`.

done